Automattic/wp-calypso
# Stats: set default view to day.

From discussion in #10465 and p5j4vm-1rU-p2, this branch changes the default stats "tab" from Insights to Day.  Furthermore, I have removed the `stats-sticky-tab` logic to ensure that all flows that lead to stats ( My Sites in masterbar, Stats in sidebar ) will open the "Day" tab initially.

The goal here is to gather usage data to compare against the current stats around Insights usage.  This will allow us to determine if we need to expose the ability to set a default stats tab, or simply keep it pointed at "Day".

__To Test__
- Open this branch locally, from the reader, click 'My Sites'.
- If a site had been selected prior, you should be viewing the Day stats page for that site.  If a prior site was not selected, you will see a Day summary for all sites.
- Change the selected site from this view, and select the "Insights" tab for this new site.  Then click the Reader link.
- Click 'My Sites' again, and note you should be shown the day tab for the site you just selected

From the My Sites section, also test by clicking to alternate stat views, Weeks/Months/ etc, then clicking to "Posts" or "Pages" in the sidebar.  Clicking the stats link in the sidebar should always take you back to 'Day'.
